npm/vite-plugin-cypress-esm/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@cypress/vite-plugin-cypress-esm",
+  "version": "0.0.0-development",
+  "description": "Make ESM Modules mutable in the browser with Cypress and Vite",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc || echo 'built, with type errors'",
+    "build-prod": "tsc || echo 'built, with type errors'",
+    "check-ts": "tsc --noEmit",
+    "cypress:open": "node ../../scripts/cypress open --project . --component",
+    "test": "node ../../scripts/cypress run --project . --browser chrome --component",
+    "watch": "tsc -w",
+    "lint": "eslint --ext .js,.ts,.json, ."
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
+    "semver": "^7.3.2"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-react": "1.3.1",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "vite": "4.1.4"
+  },
+  "files": [
+    "dist",
+    "client",
+    "index.d.ts"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cypress-io/cypress.git"
+  },
+  "homepage": "https://github.com/cypress-io/cypress/tree/develop/npm/vite-plugin-proxify-esm#readme",
+  "bugs": "https://github.com/cypress-io/cypress/issues/new?template=1-bug-report.md",
```

Comment:
nit: Can we add the `labels=npm:%20@cypress/vite-plugin-cypress-esm`label via this link?